qa/workunits/cephadm/test_iscsi_pids_limit.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+
+set -ex
+
+# checks if the containers default pids-limit (4096) is removed and Iscsi
+# containers continue to run
+# exits 1 if fails
+
+# checks which container engine is installed
+if which podman 2>&1 >/dev/null; then
+  CONT_CMD="podman"
+else
+  CONT_CMD="docker"
+fi
+
+ISCSI_CONT_IDS=$(${CONT_CMD} ps -qa --filter='name=iscsi')
```

Comment:
This fails as follows:
```
Got permission denied while trying to connect to the Docker daemon socket at unix:///var/run/docker.sock: Get http://%2Fvar%2Frun%2Fdocker.sock/v1.24/containers/json?all=1&filters=%7B%22name%22%3A%7B%22iscsi%22%3Atrue%7D%7D: dial unix /var/run/docker.sock: connect: permission denied
```
Please test both `podman` and `docker` alternatives before re-pushing.